test/testmachinery/system/shoot_cp_migration/migrate_test.go

```diff
@@ -181,12 +181,20 @@ func initGuestBookTest(ctx context.Context, t *ShootMigrationTest) (*application
 		ShootClient:       t.ShootClient,
 		SeedClient:        t.SourceSeedClient,
 	}
-	if !t.Shoot.Spec.Addons.NginxIngress.Enabled {
+	if t.Shoot.Spec.Addons == nil || t.Shoot.Spec.Addons.NginxIngress == nil || !t.Shoot.Spec.Addons.NginxIngress.Enabled {
```

Comment:
There is already a helper func that can be used. See https://github.com/gardener/gardener/blob/1f8712f7cd50b7cc023ecea12c0db75c5b9f4ec1/test/framework/applications/guestbooktest.go#L132-L134

Reply:
Done with 36f6955d